.travis.yml

```diff
@@ -1,16 +1,16 @@
 language: ruby
 
 before_install:
-  - gem i rubygems-update -v '3.0.4' && update_rubygems
+  - gem update --system '3.0.6'
```

Comment:
Do you really think we need this? I think we bumped/froze it once due to some problem. I'd hope that our recently released versions come with a good enough rubygems version. So, maybe we can drop this line entirely?

Reply:
I can try. Normally you do this when you need bug fixes from the latest rubygems, but maybe it's not necessary here. Let me remove it.

Reply:
yeah I know that's why we did it, but I can't remember when we did it anymore so I thought right now might be as good a time as any to check if we need that :grin:

Reply:
It's passing so seems good!